### DISCUSSION TITLE
Turn on captions by default setting?

### DISCUSSION THREAD

**Discussion by theontho** [FreeTubeApp/FreeTube#2122] [Q&A]
I was about to submit a feature request for this, but I see this old closed issue saying it's already in? https://github.com/FreeTubeApp/FreeTube/issues/62   Which setting is it?  I can't seem to find it, and whenever I play videos it seems to default to off.

**Comment by vallode** [comment]
Seems like #1269 implemented persistence for caption _settings_ but not for the actual captions being on or off. We should make a feature request for this. In the meantime clicking c toggles captions which I use often :)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Internal discussion

Me: Was #62 ever implemented?
P: It was, but that was before the rewrite. I guess we forgot to add it when we transitioned.

i'll reopen #62
